media-gfx/openscad/openscad-2019.05.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit elisp-common qmake-utils xdg
+
+# 2019.05
+#MY_VER=$(ver_cut 1-2) # version component
+#MY_REL=$(ver_cut 4) # release component, 'p' being the third component
+#MY_PV=${MY_VER}-${MY_REL}
+SITEFILE="50${PN}-gentoo.el"
+
+DESCRIPTION="The Programmers Solid 3D CAD Modeller"
+HOMEPAGE="http://www.openscad.org/"
+SRC_URI="https://github.com/${PN}/${PN}/releases/download/${P}/${P}.src.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="emacs"
+
+# FIXME: add optional lib3mf hidapi spnav
+RDEPEND="
+	dev-cpp/eigen:=
```

Comment:
I don't see subslots on that package. And the previous version had slot 3 defined, slot 2 was removed not long ago and you probably do not want to risk getting puzzling bug reports.